### PR TITLE
BXC-3657 - Starting applications to integration tests

### PR DESCRIPTION
**Pull request by bbpennel**
* Uses maven jetty plugin to deploy access application prior to running integration tests
* Uses maven exec plugin to start and stop solr before integration tests, since using an embedded solr instance is not accessible to the access app running in a separate jvm over in jetty
* Refreshed solrconfig.xml from the current version of solr

### CODE REVIEW

File: integration/README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Integration Tests
 
+## Running individual tests in Intellij
```
Comment (sharonluong) [inline integration/README.md:3]: do you want to add the terminal test running instructions? or is that elsewhere?

File: integration/src/test/resources/patronPrincipalConfig.json [hunk 2]
```diff
@@ -0,0 +1 @@
+  []
```
[the remark this answers is not in the document]
Comment (bbpennel) [reply integration/src/test/resources/patronPrincipalConfig.json:1]: Yes, one of the dependencies needed this json file to exist

File: integration/README.md (hunk 1 above)
Comment (bbpennel) [reply integration/README.md:3]: sure, i'll add that

File: integration/README.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+For integration tests which depend on maven to set up additional applications (such as solr or the dcr access
+application), the test needs to be run as a maven command rather than a regular junit test. To do this in Intellij,
+go to:
+
+* Run > Edit Configurations
+* Click the "+" button in the left left
```
Comment (sharonluong) [inline integration/README.md:9]: is this supposed to be top left?

File: integration/README.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+* The test can then be executed using "Run > Run...", or the test running dropdown menu in the top menu bar.
+
+## Running the tests in the terminal
+You will first need to build the project, which can be done using the following:
+`mvn clean package -DskipTests`
+
+Then to run all of the tests in the integration module, use:
+`mvn verify -pl integration`
+Or to run an individual test, use:
+`mvn -Dit.test=CollectionsEndpointIT verify -pl integration`
```
Comment (sharonluong) [inline integration/README.md:22]: you should specify where one may change the name of the test

**Review by sharonluong — APPROVED** [review]
wow, this was a doozy. nice work!

**Pull request merged**